### PR TITLE
OOP school library: set up associations

### PR DESCRIPTION
**Pull request by jevazquezb**
In this part of the project, we:

- Added the Classroom, Book, and Rental classes.
- Set up associations between classes and objects.

### CODE REVIEW

File: book.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```
Comment (rustam-tolipov) [inline book.rb:11]: Why you didn't add `add_rental(person, date)` to your book.rb?

- [ ] Adding for rental should be present `add_rental(person, date)`


For more information: [link...](`https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship`)

File: person.rb [hunk 2]
```diff
@@ -10,6 +10,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
```
Comment (rustam-tolipov) [inline person.rb:14]: - [ ] The same goes here `add_rental(book, date)`

File: tests.rb [hunk 3]
```diff
@@ -1,16 +1,73 @@
...
-capitalized_person = CapitalizeDecorator.new(person)
-p capitalized_person
-p capitalized_person.correct_name
+p class619 = Classroom.new(101)
+p class619.students
 
-capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
-p capitalized_trimmed_person
-p capitalized_trimmed_person.correct_name
+p pedro.classroom
```
Comment (rustam-tolipov) [inline tests.rb:15]: You made a great effort by adding tests.rb to your project. I absolutely liked it!

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @jevazquezb, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @jevazquezb, 




Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**